src/super_gradients/common/object_names.py

```diff
@@ -1,22 +1,3 @@
-class Losses:
```

Comment:
I don't understand why we need to remove this class...
it is not importing anything, so it cannot affect circular imports